docs/mir-snaps.md

```diff
@@ -0,0 +1,137 @@
+# Mir snaps
+
+## Introduction
+
+This document is a collection of current instructions of how to build 2 snaps
+as an example for using Mir in the days just after `snapcraft` 2.0 being
+released. The first snap contains the `mir-server`, which is the basis for what
+Canonical will eventually publish in the store and support for a given Ubuntu
+Core release. Which means, until 16.04 becomes a stable release the Mir
+version used may change, so do not plan on building any `mir-client` snaps
+until the Mir version is frozen along with the released Ubuntu archive. The
+second snap is strictly an example of a `mir-client-app`, which developers can
+use as a template to create their own. If you are experimenting and simply
+building both snaps at the same time you should experience no problems,
+however once the `mir-server` snap is in the store, it is important that the
+mir-client use the `libmirclient` version corresponding to the `libmirserver`
+within their snap.
+
+The `mir-snap` is only targeted for Ubuntu Core. It is intended for
+kiosk-like products and applications. If products require many separate UI
+applications, then they need Ubuntu Personal.
+
+This document targets Xenial Ubuntu Core and assumes your host machine is
+running Xenial (up to date) and hence using Snapcraft 2.0.
+
+> *NOTE: 2016-02-03*
+> - snapcraft 2.1 came out today and requires changes. Will followup with
+>   updates.
+> - also, the most recent Xenial kernel has [a bug creating issues for
+>   mir](https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1540731)
+
+## Instructions
+
+You need to be on a Xenial host to obtain the proper tools and the latest
+versions of stage packages. Note, one could also use `lxc` for building snaps
+in a clean environment. Also, make sure Virtual Machine Manager is installed,
+from Ubuntu Software Center or
+
+```
+apt-get install virt-manager
+```
+
+### Get the Mir Snap running first
+
+If you haven’t already, install the snapcraft tools.
+```
+$ sudo add-apt-repository ppa:snappy-dev/tools
+$ sudo apt-get update
+$ sudo apt-get upgrade
+$ sudo apt-get install snapcraft
```

Comment:
Might be worth pointing back to get-started.md rather than duplicating the instructions here.